### DISCUSSION TITLE
Input props on lambda rendering

### DISCUSSION THREAD

**Discussion by mihob** [remotion-dev/remotion#1441] [Q&A]
I am just testing the lambda rendering with your plain js HelloWorld demo. Everything is already set up and the rendering itself works fine. However it has no effect when I pass titleText as property, neither per cli nor per api. 

CLI:
`npx remotion lambda render --props='{"titleText": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit.", "titleColor":"red"}' remotion-test HelloWorld`

API:
```
await renderMediaOnLambda({
    region: "eu-central-1",
    functionName: "remotion-render-XYZ",
    composition: "HelloWorld",
    serveUrl: "https://remotionlambda-XYZ.eu-central-1.amazonaws.com/sites/remotion-test/index.html",
    codec: "h264",
    inputProps: {
        "titleText": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit.",
        "titleColor": "red"
    }
});
```
Locally everything works as expected:
`npx remotion render src/index.jsx HelloWorld out/video.mp4 --props='{"titleText": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit.", "titleColor":"red"}'`

Am I doing something wrong here? Or do I still need to customize the demo to make this work?

**Comment by JonnyBurger** [comment]
Hi @mihob! Thanks for reporting, we have just discovered that this is actually a bug in the latest release of Lambda. It looks like you use it correctly!

I have pushed a bugfix in v3.2.36. Sorry for the trouble!

**Comment by mihob** [comment]
Hey @JonnyBurger, thanks for your answer.

I have updated the package to version 3.2.36 and redeployed the function and the site. However, unfortunately, it still does not work. Is there anything else to do?

**Reply by JonnyBurger** [reply]
Hey @mihob, apologies, I indeed found another bug! I have released v3.2.37 now. Let me know if it works and sorry for the botched update!

**Reply by mihob** [reply]
@JonnyBurger Thx for your support! Now everything looks fine!
